src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2774,7 +2978,7 @@ because xbuild doesn't support framework reference assemblies.
 	<RemoveDirFixed Directories="$(MonoAndroidIntermediate)proguard" Condition="Exists ('$(MonoAndroidIntermediate)proguard')" />
 	<RemoveDirFixed Directories="$(MonoAndroidIntermediateResourceCache)" Condition="Exists ('$(MonoAndroidIntermediateResourceCache)')" />
 	<RemoveDirFixed Directories="$(_AndroidAotBinDirectory)" Condition="Exists ('$(_AndroidAotBinDirectory)')" />
-	<Delete Files="$(IntermediateOutputPath)_dex_stamp" />
+	<Delete Files="$(MonoAndroidIntermediate)_dex_stamp" />
```

Comment:
This should not be changed; the `_CompileToDalvikWithDx` target creates `$(IntermediateOutputPath)_dex_stamp`.